### ISSUE TITLE
issue with miner in ubuntu

### ISSUE THREAD

**Issue by dragonfly62** [andru-kun/wildrig-multi#142]
here is the messge I got when running the driver in ubuntu " error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory "

**Comment by ahorek** [comment]
yes, these libraries (libssl and libcrypto 1.1) are no longer available in public repositories, but it's still possible to install them manually.

it would be great if the miner could support openssl 3 out of the box. https://discord.com/channels/484987277596950538/484987277596950540/983000010926985276

**Comment by dragonfly62** [comment]
it does not look trivial to me , how would you do that?

**Comment by ahorek** [comment]
https://www.youtube.com/watch?v=YuJPpCfuSJ0 or https://linuxpip.org/install-openssl-linux/ should do the trick

**Comment by dragonfly62** [comment]
thank you that made the trick
